src/main/java/uk/ac/cam/cl/dtg/isaac/api/AssignmentFacade.java

```diff
@@ -441,6 +442,9 @@ public Response getAssignmentProgressDownloadCSV(@Context final HttpServletReque
                     currentlyLoggedInUser.getFamilyName()));
 
             List<String> headerRow = Lists.newArrayList(Arrays.asList("", ""));
+            if (includeUserIDs) {
+                headerRow.add("");
```

Comment:
Is it clear in the CSV that this is the user ID?

Reply:
yeah, there's a title

Reply:
This is the header row of the question titles, which is offset by the number of name/user ID columns. You'll see the two empty cells added on line 444, if we also have user ID, we need three empty cells offset!